$OpenBSD: patch-contrib_hdtbl_Makefile_sub,v 1.1 2011/03/19 16:48:53 schwarze Exp $
# These examples use bash at build time, but i don't want a bash dependency.
--- contrib/hdtbl/Makefile.sub.orig	Sun Mar 13 18:14:15 2011
+++ contrib/hdtbl/Makefile.sub	Sun Mar 13 18:19:24 2011
@@ -47,34 +47,11 @@ STRIPFILES=\
   hdmisc.tmac
 
 # These files are handled by the `.in.roff' rule.
-GENFILES=\
-  examples/fonts_n.roff \
-  examples/fonts_x.roff
+GENFILES=
 
-EXAMPLEFILES=\
-  examples/common.roff \
-  examples/chess_board.roff \
-  examples/color_boxes.roff \
-  examples/color_nested_tables.roff \
-  examples/color_table_cells.roff \
-  examples/color_transitions.roff \
-  examples/col_rowspan_colors.roff \
-  examples/mixed_pickles.roff \
-  examples/rainbow.roff \
-  examples/short_reference.roff
+EXAMPLEFILES=
 
-PROCESSEDEXAMPLEFILES=\
-  examples/chess_board.ps \
-  examples/color_boxes.ps \
-  examples/color_nested_tables.ps \
-  examples/color_table_cells.ps \
-  examples/color_transitions.ps \
-  examples/col_rowspan_colors.ps \
-  examples/fonts_n.ps \
-  examples/fonts_x.ps \
-  examples/mixed_pickles.ps \
-  examples/rainbow.ps \
-  examples/short_reference.ps
+PROCESSEDEXAMPLEFILES=
 
 EXAMPLEFILES_=`echo $(EXAMPLEFILES) | sed 's|examples/||g'`
 GENFILES_=`echo $(GENFILES) | sed 's|examples/||g'`
